### ISSUE TITLE
1.20.1

### ISSUE THREAD

**Issue by MinecraftGuy926** [Urokhtor/container-tooltips#1]
Would it be possible to backport to 1.20.1 please? Love this idea

**Comment by Urokhtor** [comment]
Thanks!

I've actually been thinking what Minecraft versions I should support, but since this is a brand new mod, I figured out it's better to wait and see if anyone actually wants to use this mod.

I already looked at supporting 1.20.1 at some point. There was some incompatibility between that version and 1.20.4, but it's probably possible to fix that issue. I can take a look.

**Comment by Urokhtor** [comment]
Managed to get the mod ported to Minecraft 1.20.1. It was mostly rather straightforward, but the GUI texture rendering has changed since that Minecraft version. It's easier to draw the inventory background in newer versions since the game does it largely for you. But 1.20.1 does not have such utility, so I had to take the bundle texture and draw the inventory slots without drawing a separate background texture underneath. That's why the background looks slightly different with 1.20.1 version of this mod. But functionally it is the same.

It's already released on Modrinth, so feel free to start playing. :)

a894c51809db28b09cd6bf94cd1ddee338efca93
